awx/main/tasks/receptor.py

```diff
@@ -364,26 +369,19 @@ def _run_internal(self, receptor_ctl):
                     logger.warn(f"Could not launch pod for {log_name}. Exceeded quota.")
                     self.task.update_model(self.task.instance.pk, status='pending')
                     return
-                # If ansible-runner ran, but an error occured at runtime, the traceback information
-                # is saved via the status_handler passed in to the processor.
-                if state_name == 'Succeeded':
-                    return res
```

Comment:
There's an in-congruence here. Yes, if the status_handler received this information the receptor output should be this... but it's an imperfect metric. These two things are associated, but other events could cause one to be true but not the other.

so I forced it by doing this

```diff
diff --git a/awx/main/tasks/callback.py b/awx/main/tasks/callback.py
index ab27ce9f36..fd720ac985 100644
--- a/awx/main/tasks/callback.py
+++ b/awx/main/tasks/callback.py
@@ -220,6 +220,8 @@ class RunnerCallback:
             result_traceback = status_data.get('result_traceback', None)
             if result_traceback:
                 self.delay_update(result_traceback=result_traceback)
+        elif status_data['status'] == 'successful':
+            self.delay_update(result_traceback='alan set this!')
 
 
 class RunnerCallbackForProjectUpdate(RunnerCallback):
```

And this works as I expected. The new `if` condition is accurate. I like this also because it avoids running additional receptor commands when they are not necessary. That's something the `if` here was accomplishing before, and I didn't want to lose that.